services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]